synapse/handlers/federation.py

```diff
@@ -1705,8 +1719,31 @@ async def _sync_partial_state_room_wrapper() -> None:
                     room_id=room_id,
                 )
             finally:
+                # Check whether the room is still partial stated, while we still claim
+                # to be the active sync. Usually, the partial state flag will be gone,
+                # unless we left and rejoined the room, or the sync failed.
+                is_still_partial_state_room = await self.store.is_partial_state_room(
+                    room_id
+                )
```

Comment:
To check: is the meaning of this flag (currently) "there is some event in this room whose full state-before has not been computed"?

Reply:
er, sure.
The flag doesn't imply that there's a running partial state sync.